shell/platform/android/io/flutter/plugin/editing/ListenableEditingState.java

```diff
@@ -179,6 +257,54 @@ public SpannableStringBuilder replace(
       Log.e(TAG, "editing state should not be changed in a listener callback");
     }
 
+    Log.e(
```

Comment:
Weill `replace` be called when the engine receives a text change from the framework?

Reply:
Yes replace is called when the engine receives a text change. It replaces the entire value with the one from the framework.

Reply:
Is it useful to setDeltas in that case?  Could that delta ever end up being sent back to the framework when it shouldn't be?